package.json

```diff
@@ -98,5 +98,8 @@
   "bugs": {
     "url": "https://github.com/oaknational/oak-components/issues"
   },
-  "homepage": "https://github.com/oaknational/oak-components#readme"
+  "homepage": "https://github.com/oaknational/oak-components#readme",
+  "overrides": {
+    "jsdom": "24.0.0"
```

Comment:
The version of JSDOM pinned by `jest-environment-jsdom` is very old and doesn't support `aspect-ratio` (which has been around for a very long time). This overrides it with the latest version.